app/packages/files/file.ts

```diff
@@ -158,14 +158,10 @@ export default abstract class File {
         await this.fileModel.rename(newName, conflict);
     }
 
-    @task
-    @waitFor
```

Comment:
Why do we want to make this a native `async` function instead of a e-c task?

Reply:
I turned this back into a normal async function, as we weren't using any of the ember-concurrency features for it, but I can turn it back into a e-c task